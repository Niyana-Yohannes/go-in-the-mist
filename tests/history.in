history
place A1
history
place A2
place A3
place A4
history
term
