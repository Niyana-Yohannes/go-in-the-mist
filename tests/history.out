
A1
A1A2A3A4
